### PR TITLE
Change report generation call

### PR DESCRIPTION
**Pull request by boulund**
Ignore additional arguments to snakemake when creating report,

Intended to avoid shell escaping issue with complex arguments to e.g. Singularity, issue #160

### CODE REVIEW

**Comment by AroArz** [comment]
Did you have a chance to test this?

**Comment by boulund** [comment]
Haha. Not at all, just made the changes in the github web editor. Don't have an opportunity to test this today.

**Review by AroArz — CHANGES_REQUESTED** [review]
Okay so apparently `snakemake_call` returns the entire path to snakemake like `/home/aroarz/miniconda3/bin/snakemake --use-singularity --singularity-args -B /proj/snic2020-6-233/db`.

Which means `snakemake_call[0]` returns `/`, still producing an error.

File: Snakefile [hunk 1]
```diff
@@ -153,10 +153,8 @@ onsuccess:
             Path("citations.rst").unlink()
         Path("citations.rst").symlink_to(citation_filename)
 
-        snakemake_call = " ".join(argv)
-        shell("{snakemake_call[0]} --unlock".format(snakemake_call=snakemake_call))
-        shell("{snakemake_call[0]} --report {report}-{datetime}.html".format(
-            snakemake_call=snakemake_call,
+        shell("snakemake --unlock")
```
Comment (boulund) [inline Snakefile:156]: How about using this instead then?
```suggestion
        shell("{argv[0]}--unlock")
```

File: Snakefile [hunk 2]
```diff
@@ -153,10 +153,8 @@ onsuccess:
...
+        shell("snakemake --report {report}-{datetime}.html".format(
```
Comment (boulund) [inline Snakefile:157]:
```suggestion
        shell("{snakemake_call} --report {report}-{datetime}.html".format(
                snakemake_call=argv[0],
```

File: Snakefile (hunk 1 above)
Comment (AroArz) [reply Snakefile:156]: This works

File: Snakefile (hunk 2 above)
Comment (AroArz) [reply Snakefile:157]: This also works, we can commit those changes and merge the PR

**Review by AroArz — APPROVED** [review]

**Comment by boulund** [comment]
Did you test it already? You're so quick!

**Comment by AroArz** [comment]
Yes! I tested all the changes, I also added the bit to the changelog

**Comment by boulund** [comment]
Awesome!

**Pull request merged**